shared/ui/testing/src/main/java/com/ivy/ui/testing/PaparazziScreenshotTest.kt

```diff
@@ -4,9 +4,17 @@ import androidx.compose.runtime.Composable
 import app.cash.paparazzi.DeviceConfig
 import app.cash.paparazzi.Paparazzi
 import com.ivy.design.system.IvyMaterial3Theme
+import org.junit.Before
 import org.junit.Rule
+import java.util.Locale
 
 open class PaparazziScreenshotTest {
+
+    @Before
```

Comment:
@ILIYANGERMANOV I added this snippet to ensure that paparazzi doesn’t rely on local settings, which were causing issues with screen registration. This should standardize the tests and prevent any local settings from affecting them.

Reply:
Perfect 💯